packages/cli/src/commands/hydrogen/init.ts

```diff
@@ -66,12 +67,16 @@ export default class Init extends Command {
   };
 
   async run(): Promise<void> {
-    const {flags} = await this.parse(Init);
+    // Rename markets => i18n
+    const {
+      flags: {markets, ..._flags},
+    } = await this.parse(Init);
+    const flags = {..._flags, i18n: markets};
 
-    if (flags.markets && !I18N_CHOICES.includes(flags.markets as I18nChoice)) {
+    if (flags.i18n && !I18N_CHOICES.includes(flags.i18n as I18nChoice)) {
```

Comment:
Why rename it to `i18n` here? Do we also want to change the actual CLI option to be `i18n`?

Reply:
It was i18n originally, and it's like that in the rest of the code. We changed the public flag to "markets" at the very end since it will be useful later on to call it that way (once we connect better to the actual Markets).